### ISSUE TITLE
[chassis] [wolverine card] syncd busy

### ISSUE THREAD

**Issue by wenyiz2021** [aristanetworks/sonic#87]
Hi @Staphylo, @kartik, @kenneth-arista 

on the wolverine card, syncd is still busy and causing CPU usage high consistently
especially, idle CPU usage per core is also high on wolverine card, compared to the other 2 downlink cards, check (100% - column 4) to get idle CPU usage:

uplink wolverine:
```
%Cpu0  : 10.7 us,  0.7 sy,  0.0 ni, **88.7 id,**  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu1  : 21.9 us,  0.3 sy,  0.0 ni, 77.7 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu2  : 14.7 us,  1.3 sy,  0.0 ni, 83.6 id,  0.0 wa,  0.0 hi,  0.3 si,  0.0 st
%Cpu3  : 37.1 us,  0.3 sy,  0.0 ni, 62.6 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu4  :  7.4 us,  1.7 sy,  0.0 ni, 91.0 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu5  :  7.0 us,  0.3 sy,  0.0 ni, 92.6 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu6  : 28.5 us,  0.3 sy,  0.0 ni, 70.9 id,  0.3 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu7  : 30.8 us,  0.0 sy,  0.0 ni, 69.2 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
```
downlink CW2:
```
%Cpu0  :  8.3 us,  0.3 sy,  0.0 ni, 91.4 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu1  : 15.1 us,  0.7 sy,  0.0 ni, 84.3 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu2  : 10.3 us,  1.7 sy,  0.0 ni, 88.1 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu3  : 11.3 us,  0.3 sy,  0.0 ni, 88.4 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu4  : 19.6 us,  0.3 sy,  0.0 ni, 80.1 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu5  :  0.3 us,  0.3 sy,  0.0 ni, 99.3 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu6  :  3.6 us,  0.3 sy,  0.0 ni, 96.0 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
%Cpu7  :  4.3 us,  1.0 sy,  0.0 ni, 94.7 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
```

checked another vendor chassis have uplink card syncd process less busy, idle CPU usage per core <10%

1. any idea if uplink card syncd process could be busier than downlink cards?
2. does # of bgp neighbors, routes, links has an impact on syncd process?

basically want to know if there is real issue on WV card

@arlakshm for viz

**Comment by arlakshm** [comment]
admin@str2-7804-lc5-1:~$ top -H
top - 21:52:47 up  7:26,  1 user,  load average: 1.28, 1.58, 2.02
Threads: 986 total,   3 running, 977 sleeping,   0 stopped,   6 zombie
%Cpu(s): 16.7 us,  0.3 sy,  0.0 ni, 83.0 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
MiB Mem :  15918.6 total,   4471.0 free,   7074.7 used,   4372.9 buff/cache
MiB Swap:      0.0 total,      0.0 free,      0.0 used.   8445.5 avail Mem

    PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND
 473988 root      20   0 4700148   1.1g 189852 S  32.5   7.2   6:57.93 bcmLINK.0
 473976 root      20   0 4699968   1.1g 189856 R  32.1   7.2   6:57.69 bcmLINK.0
 473967 root      20   0 4699968   1.1g 189856 S  24.5   7.2   4:26.31 bcmCNTR.0
 473972 root      20   0 4700148   1.1g 189852 R  20.9   7.2   4:38.04 bcmCNTR.0
 475108 root      20   0 4700148   1.1g 189852 S   6.3   7.2   0:45.15 DnxCtrBg
 475043 root      20   0 4699968   1.1g 189856 S   6.0   7.2   0:48.85 DnxCtrBg
 475229 root      20   0 4700148   1.1g 189852 S   4.0   7.2   0:17.21 syncd
   2289 root      20   0  268024  58460   7864 S   1.7   0.4  55:58.66 redis-server
 473655 root      20   0  141044  52140  14180 S   1.7   0.3   1:02.37 python3
   2287 root      20   0  302840  78768   7540 S   1.3   0.5  39:21.20 redis-server
 475131 root      20   0 4699968   1.1g 189856 S   0.7   7.2   0:11.84 syncd
 502540 admin     20   0   11908   4672   3128 R   0.7   0.0   0:01.41 top
   1502 root      20   0   75512  21140   7752 S   0.3   0.1   1:38.08 redis-server
 471557 root      20   0   47692  35152  13540 S   0.3   0.2   0:00.79 chassisd
 475810 root      20   0  271040  39916  13992 S   0.3   0.2   0:02.17 python3
 472672 root      20   0   29600  25324   9376 S   0.3   0.2   0:12.83 supervisord
 475185 root      20   0 4699968   1.1g 189856 S   0.3   7.2   0:00.66 syncd
 475227 root      20   0 4700148   1.1g 189852 S   0.3   7.2   0:11.19 syncd
 473536 root      20   0 1620452  85736  19616 S   0.3   0.5   0:00.72 telemetry
 473634 root      20   0 1620452  85736  19616 S   0.3   0.5   0:00.44 telemetry
 473753 root      20   0 1616176  85464  16876 S   0.3   0.5   0:00.53 dialout_client_
 473823 root      20   0 1616176  85464  16876 S   0.3   0.5   0:00.32 dialout_client_
      1 root      20   0  168040  14624   8996 S   0.0   0.1   1:58.88 systemd
      2 root      20   0       0      0      0 S   0.0   0.0   0:00.03 kthreadd
      3 root       0 -20       0      0      0 I   0.0   0.0   0:00.00 rcu_gp

**Comment by kenneth-arista** [comment]
@wenyiz2021 is this still an outstanding issue? In two testbeds with Wolverine as their uplink linecards, I see each syncd process is varying between 50-60% CPU when sonic-mgmt tests are not running. What conditions are you testing when looking at top?

**Comment by wenyiz2021** [comment]
this is ok now, I increased memory threshold in test for worlverine hwsku to 90%, it was 60 so sometimes it''' still fail the test.
The reason for increasing this is that I also see other chassis occasionally fail. so I'm thinking it is ok.
also I'm more focusing on cpu usage per core, using `top` then enter `1`, so less variant.
